### ISSUE TITLE
Possibility to replicate the number of geth

### ISSUE THREAD

**Issue by noeRls** [skillz-blockchain/volt_entity#32]
Here is a list of problems i have in mind:
- [ ] How do you sync two geth not on the same cluster. Expose the pod's ip? Make custom routes for each of pods to provide them in https?

**Comment by lucasclerisse** [comment]
Opening a new route seems to me to be the most interesting because it allows us to be perfectly uniform according to the protocol. It also allows us to add abstraction to our implementation.

**Comment by noeRls** [comment]
I did some research on it. The protocol used by geth to comunicate between them is telnet so you can't have custom route like `ip/geth1`, you need to have a dedicated ip/dns. In order to do that we can use a loadbalancer and k8s provide us a dns. The problem is that we need to have a loadbalancer by replica and i have no idea of how to do that. Arnaud talked about "StatefulSet". I'm going to do some research on it. In addition the api need to be able to connect to each pod individually (in case of add peer for example).
To sum up we're searching to __have a service by replica__, if we can do that then it will be pretty easy to implement it _(snapshot system excluded)_.
